srcpkgs/libgbinder/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'libgbinder'
 pkgname=libgbinder
-version=1.1.26
+version=1.1.27
```

Comment:
1.1.28 available

Reply:
Updated

Reply:
and now .29 is available 🙃

Reply:
I guess this is better than the projects that release no new tags for 2 years lol, updated to .29 and tested as working (with Waydroid)